src/explore-education-statistics-admin/src/services/types/content.ts

```diff
@@ -11,6 +11,13 @@ export interface Comment {
   };
   created: string;
   updated?: string;
+  resolved?: string;
+  resolvedBy?: {
```

Comment:
As we don't use the "id" or "email" field, you don't need to include them here.  We should I guess not really be returning them from the backend ideally if they're not going to be used, as they leak potentially sensitive information (see my other comment about switching to returning UserViewModels)

Reply:
As discussed on standup, I'm leaving this as it is for now until we decide whether to split the resolved functionality into a separate endpoint.